activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryTasksIT.java

```diff
@@ -569,13 +569,16 @@ public void shouldGetAddRemoveTaskGroupCandidates() {
 
         //when
         ResponseEntity<List<String>> responseEntity = getCandidateGroups(createdTask.getId());
+        ResponseEntity<Task> taskResponseEntity = executeRequestGetTasksById(createdTask.getId());
 
         //then
         assertThat(responseEntity).isNotNull();
         assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
         assertThat(responseEntity.getBody()).isNotNull();
         assertThat(responseEntity.getBody().size()).isEqualTo(1);
-        assertThat(responseEntity.getBody().get(0)).isEqualTo("testgroup");
+        assertThat(responseEntity.getBody()).containsExactly("testgroup");
+        assertThat(taskResponseEntity.getBody().getCandidateGroups()).hasSize(1);
```

Comment:
Tip: `containsExactly` is already checking the size, so you could remove this assert if you want.

Reply:
good advice, thanks